docs/pages/blog/mui-product-comparison.md

```diff
@@ -0,0 +1,190 @@
+---
+title: An introduction to the MUI ecosystem
+description: MUI is more than just Material UI. Consider Joy UI, MUI Base, MUI X, and Toolpad for your next project.
+date: 2022-10-24T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
+tags: ['Product']
+---
+
+<img src="/static/blog/mui-product-comparison/ecosystem.png" style="width: 692px; margin-bottom: 24px; aspect-ratio: 173/80" alt="Logos for all the high-level MUI products." />
+
+MUI is so much more than Material UI!
+
+You'll be forgiven if you thought MUI was merely shorthand for our most popular product.
+What began as an open-source React UI library has evolved over time into a full-fledged startup with a whole suite of related products.
```

Comment:
Do describe a stronger expansion of scope?

```suggestion
What began as an open-source React UI library that implements Material Design has evolved over time into a full-fledged startup with a whole suite of related products to build UI faster.
```

Reply:
I think these details are made clear enough in the next couple lines.

Reply:
What got me confused is that "what started at X" but we are still mostly X.

Reply:
I think I see the source of the confusion, which is that Material UI didn't _become_ MUI—instead, MUI _grew out of_ Material UI. So it's not really correct to say that the "library" has evolved into a "startup."

How about this instead? I think it conveys the same idea more clearly, without repeating what's already stated elsewhere in the intro.

```suggestion
In fact, MUI as an organization was founded to meet the growing needs of the Material UI community, and is now responsible for a whole suite of related products.
```